Build/Run-Tests.ps1

```diff
@@ -5,18 +5,26 @@ $packageConfig = [xml](Get-Content .nuget\packages.config)
 $opencover_version = $packageConfig.SelectSingleNode('/packages/package[@id="OpenCover"]').version
 $opencover_console = "packages\OpenCover.$opencover_version\tools\OpenCover.Console.exe"
 
-&$opencover_console `
-    -register:administrator `
-    -returntargetcode `
-    -hideskipped:All `
-    -filter:"+[*]* -[FluentAssertions*]* -[SmartFormat*]* -[nunit*]*" `
-    -excludebyattribute:*.ExcludeFromCodeCoverage* `
-    -excludebyfile:*\*Designer.cs `
-    -output:"OpenCover.GitExtensions.xml" `
-    -target:"nunit3-console.exe" `
-    -targetargs:"$testAssemblies --agents=1 --workers=1 --timeout=90000"
-$testExitCode = $LastExitCode
-Push-AppveyorArtifact "TestResult.xml"
+$testRunCount = 10
```

Comment:
❗️ This should be 1 by default

```suggestion
$testRunCount = 1
```

Reply:
This commit "Temporarily ..." will be reverted. This PR is marked "WIP" and labeled "experimental".

Reply:
This file has become obsolete with the Arcade build.
Any objections to remove it?